core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/request/MixedMetricsRequest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics.core.impl.request;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hawkular.metrics.core.api.Availability;
+import org.hawkular.metrics.core.api.Gauge;
+
+import com.google.common.base.Objects;
+import com.wordnik.swagger.annotations.ApiModel;
+
+/**
+ * @author Stefan Negrea
+ */
+@ApiModel
+public class MixedMetricsRequest {
```

Comment:
This is not the only stray class in that package, there is at least another request class. I would like to defer moving this into the JAXRS API package until the core and impl are refactored.